scripts/data_files/query_config.fmt

```diff
@@ -99,6 +99,10 @@
 #define MACRO_NAME_TO_STR(macro)                                        \
     mbedtls_printf( "%s", strlen( #macro "" ) > 0 ? #macro "\n" : "" )
 
+#define NAME_TO_STR(macro)  #macro
+#define OUTPUT_MACRO_NAME_VALUE(macro) mbedtls_printf( #macro "%s\n",   \
+    strlen( NAME_TO_STR(macro) "") > 0 ? "=" NAME_TO_STR(macro) : "" )
```

Comment:
Minor: I suggest `NAME_TO_STR(macro)[0] != 0` rather than using `strlen` to test for emptiness. It saves a call when stepping through code in a debugger. Not really important in this particular program though.